client/src/main/java/com/vesoft/nebula/client/meta/AbstractMetaClient.java

```diff
@@ -31,8 +31,7 @@ public AbstractMetaClient(List<HostAddress> addresses, int timeout,
             String host = address.getHost();
             int port = address.getPort();
             // check if the address is a valid ip address or uri address and port is valid
-            if ((!InetAddresses.isInetAddress(host) || !InetAddresses.isUriInetAddress(host))
-                || (port <= 0 || port >= 65535)) {
+            if (port <= 0 || port >= 65535) {
```

Comment:
You can use InetAddresses to check the ip or hostname is valid.

Reply:
URI checking is not needed. let developer use any name he/she wants. It could be persisted on server /etc/hosts or could be resolved by DNS. He should be able to use any name

Reply:
I mean to  change to check if the IP or domain name is valid.